addons/mass_mailing/views/mailing_list_views.xml

```diff
@@ -120,19 +120,30 @@
                 <group>
                     <field name="is_public"/>
                 </group>
+            </form>
+        </field>
+    </record>
+
+    <record id="mailing_list_view_form_simplified_footer" model="ir.ui.view">
+        <field name="name">mailing.list.form.simplified.footer</field>
```

Comment:
By the way, maybe we should set a priority on ``mailing_list_view_form_simplified`` also in order to ensure default view is always mailing_list_view_form ?

Reply:
@tde-banana-odoo Good point, it seems safer. Going for 10 on simplified and 20 on simplified_footer for instance?

Reply:
Well default priority is 16 so pay attention when giving values ^^

Reply:
I just found that, sorry. 25 and 30 should work then :)